src/orchestrator/stubs/__init__.py

```
@@ -125,16 +128,48 @@
+        )
+
+
+class ArchitectStub(StubGraph[ArchitectState]):
+    def __init__(
+        self,
+        *,
+        agent_config: Optional[AgentConfiguration] = None,
+        checkpointer: Optional[Checkpointer] = None,
+        store: Optional[BaseStore] = None,
+    ):
+        print("CRFEATING ARCHITECT")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)